.github/pull_request_template.md

```diff
@@ -1,19 +1,19 @@
 Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 
 
-> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double check the file for other mistakes in order to fix all the issue on the current cheat sheet.
+> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.
 
 Please make sure that for your contribution:
 
 - [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
-- [ ] All the markdown files do not raise any validation policy violation, see policy [here](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
+- [ ] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
 - [ ] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries#conversion-rules).
 - [ ] All your assets are stored in the **assets** folder.
 - [ ] All the images used are in the **PNG** format.
-- [ ] Any references to website have been formatted as [TEXT](URL)
-- [ ] You verified/tested the effectiveness of your contribution (e.g.: defensive code proposed is really an effective remediation? Please verify it works!).
+- [ ] Any references to the website have been formatted as [TEXT](URL)
+- [ ] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
 - [ ] The CI build of your PR pass, see the build status [here](https://travis-ci.org/OWASP/CheatSheetSeries/pull_requests).
 
-If your PR is related to an issue. Please end your PR text with the following line:
+If your PR is related to an issue. Please finish your PR text with the following line:
```

Comment:
This should probably be one sentence with a comma between the two parts - otherwise the first is just a fragment that doesn't make sense.

Reply:
Indeed.